.yamato/mobile-build-and-test.yml

```diff
@@ -2,18 +2,15 @@
   name: 2021.2 Build Android player
   agent:
       type: Unity::VM
-      image: package-ci/win10:stable
+      image: mobile/android-execution-r19:v0.0.2-546036
       flavor: b1.large
   commands:
     - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
     - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output utr.bat
```

Comment:
I'm guessing this image is not windows, so need to drop the batch file stuff

Reply:
It actually is a Windows image.